backend/oasst_backend/prompt_repository.py

```diff
@@ -39,6 +39,9 @@
 from sqlalchemy.orm.attributes import flag_modified
 from sqlmodel import JSON, Session, and_, func, literal_column, not_, or_, text, update
 
+# TODO: Add doc string to all methods and classes.
```

Comment:
I don't think we should have random TODOs like this, if we want to embark on adding docstrings throughout the code we should raise a GitHub issue for it instead